commit bf6f1276768f31bdd00c9d5f50cff4c26330a101
Author: Christian Hewitt <christianshewitt@gmail.com>
Date:   Wed Nov 21 13:39:29 2018 +0400

    drm/meson: add support for 1080p25 mode
    
    [ Upstream commit 31e1ab494559fb46de304cc6c2aed1528f94b298 ]
    
    This essential mode for PAL users is missing, so add it.
    
    Fixes: 335e3713afb87 ("drm/meson: Add support for HDMI venc modes and settings")
    Signed-off-by: Christian Hewitt <christianshewitt@gmail.com>
    Acked-by: Neil Armstrong <narmstrong@baylibre.com>
    Signed-off-by: Neil Armstrong <narmstrong@baylibre.com>
    Link: https://patchwork.freedesktop.org/patch/msgid/1542793169-13008-1-git-send-email-christianshewitt@gmail.com
    Signed-off-by: Sean Paul <seanpaul@chromium.org>
    Signed-off-by: Sasha Levin <sashal@kernel.org>

diff --git a/drivers/gpu/drm/meson/meson_venc.c b/drivers/gpu/drm/meson/meson_venc.c
index 14aac661f38b..7a3a6ed9f27b 100644
--- a/drivers/gpu/drm/meson/meson_venc.c
+++ b/drivers/gpu/drm/meson/meson_venc.c
@@ -715,6 +715,7 @@ struct meson_hdmi_venc_vic_mode {
 	{ 5, &meson_hdmi_encp_mode_1080i60 },
 	{ 20, &meson_hdmi_encp_mode_1080i50 },
 	{ 32, &meson_hdmi_encp_mode_1080p24 },
+	{ 33, &meson_hdmi_encp_mode_1080p50 },
 	{ 34, &meson_hdmi_encp_mode_1080p30 },
 	{ 31, &meson_hdmi_encp_mode_1080p50 },
 	{ 16, &meson_hdmi_encp_mode_1080p60 },
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1090_bf6f1276768f31bdd00c9d5f50cff4c26330a101.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1090_bf6f1276768f31bdd00c9d5f50cff4c26330a101.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
